lib/BitbucketHandler.js

```diff
@@ -460,11 +461,14 @@ BitbucketHandler.prototype.authLookupController = function(req, res, next) {
       },
     };
 
+    var self = this;
     var bitbucket = this.getBitbucketApi(project);
     bitbucket.refreshToken(function(err, auth) {
       if (err) {
         return next(err);
       }
+      self.log.info({auth: auth}, 'refreshed auth token from BB');
```

Comment:
I would personally rather not store the token in the logs. I know with BB they are short-lived, but they could still be stolen ... if we want to have something traceable in the logs, maybe we could store the user ID? or something similar ...